tests/test_flatten.py

```diff
@@ -8,6 +11,9 @@
         "contextInfo": {"context1": "info1", "context2": "info2"},
     }
 ]
+PD_DATA = DataFrame(DICT_DATA)
+GEOMETRY = [Point(4, 2)]
```

Comment:
Althou the idea is to start working with DataFram, I have already added those constants, so after finishing working with DataFrame, I could add GeoDataFrame support also.

Reply:
So should you update the PR description saying this PR focus on both (dataframe and geodataframe)?

Reply:
indeed.